pkg/backup/resource_backupper.go

```diff
@@ -222,8 +222,10 @@ func (rb *defaultResourceBackupper) backupResource(group *metav1.APIResourceList
 		}
 
 		var labelSelector string
+		excludeLabelSelector := "velero.io/exclude-from-backup!=true"
+		labelSelector = excludeLabelSelector
 		if selector := rb.backupRequest.Spec.LabelSelector; selector != nil {
-			labelSelector = metav1.FormatLabelSelector(selector)
+			labelSelector = labelSelector + "," + metav1.FormatLabelSelector(selector)
```

Comment:
I would `s/labelSelector/"velero.io/exclude-from-backup!=true"` and delete lns 225,226. WDYT? I don't think this is so complex that needs to be expanded. Less lines is better if it's still clear.

Reply:
I'd recommend:

```go
labelSelector := "velero.io/exclude-from-backup!=true"
if selector := rb.backupRequest.Spec.LabelSelector; selector != nil {
    labelSelector = labelSelector + "," + metav1.FormatLabelSelector(selector)
...
```

Reply:
Agree there's a few more lines than necessary here, I like @skriss's suggestion as it will make it easier to find just that labelSelector in the future